phoenix-core/src/test/java/org/apache/phoenix/util/IndexUtilTest.java

```diff
@@ -138,6 +138,10 @@ public void testIndexNullableColumnDataType() {
     private void verifyIndexColumnDataTypes(boolean isNullable, String expected) {
         Map<String, String> indexColumnDataTypes = Maps.newTreeMap();
         for (PDataType dataType : PDataTypeFactory.getInstance().getTypes()) {
+            if (!dataType.isEqualitySupported()) {
```

Comment:
I thought we will support creating indexes on JSON columns ?

Reply:
No we don't support indexes on JSON column types, but we can support indexes on scalar values from within JSON objects. eg:
CREATE INDEX my_index ON my_table (json_value(json_col, '$.firstName'),
json_value(json_col, '$.address.city')) INCLUDE json_value(json_col, '$.lastName');